src/raftstore/store/snap.rs

```diff
@@ -254,9 +255,12 @@ mod v1 {
     /// `save` method to make them persistent. When saving a crc32 checksum
     /// will be appended to the file end automatically.
     pub struct Snap {
+        mgr: SnapManager,
```

Comment:
Rare, but I did meet that. I'll take a few minus to understand why the manager is need.